Common/include/CConfig.hpp

```diff
@@ -1141,6 +1159,37 @@ class CConfig {
   unsigned short Kind_Inlet_InterpolationType;    /*!brief type of spanwise interpolation data to use for the inlet face. */
   bool PrintInlet_InterpolatedData;               /*!brief option for printing the interpolated data file. */
 
+  /* other NEMO configure options*/
```

Comment:
Just a heads up regarding these NEMO configure options will exist within the a future fluid model addition.